modules/kelbo.py

```diff
@@ -0,0 +1,30 @@
+from .base import Module
+import random
+import re
+
+class Kelbo(Module):
+    DESCRIPTION = '____ __ _____'
+    ARGC = 1
+
+    # returns a random string of ___ __ _ ____
+    def kelbo1(self):
+      num_of_groups = random.randint(2, 10)
+      kelboed_message = ""
+      while num_of_groups > 0:
+        kelboed_message+="_"*random.randint(1, 5)
+        kelboed_message+=" "
+        num_of_groups-=1
+      return kelboed_message
+
+    # returns a kelboified version of a text arg
+    def kelbo2(self, text):
+      kelboed_message = re.sub("[^\s]", "_", text)
```

Comment:
Hmm. While I love `re` as much as anyone, I feel like an easier solution might be to iterate through each character and replace it unless it's a space. You could even do it with a list comprehension if you like:
```py
return "".join([" " if character == " " else "_" for character in text])
```